pkg/controller/plan/adapter/ovirt/builder.go

```diff
@@ -358,6 +368,10 @@ func (r *Builder) mapCPU(vm *model.Workload, object *cnv.VirtualMachineSpec) {
 		Cores:   uint32(vm.CpuCores),
 		Threads: uint32(vm.CpuThreads),
 	}
+	if vm.CpuPinningPolicy == Dedicated {
+		object.Template.Spec.Domain.CPU.DedicatedCPUPlacement = true
```

Comment:
Although I added only this value, we can also isolate the QEMU emulator. We also may combine it with QoS for CPU and/or [`CPUFeature`](https://kubevirt.io/api-reference/master/definitions.html#_v1_cpufeature).

Reply:
IIRC we don't isolate the emulator thread or change QoS/CPU features in ovirt when setting the dedicated CPUs policy, no?

Reply:
> IIRC we don't isolate the emulator thread or change QoS/CPU features in ovirt when setting the dedicated CPUs policy, no?

yes, we don't change any of them. the only close thing to the emulator is what we take for VDSM. it's not the same of course.